strings_lab.js

```diff
@@ -0,0 +1,164 @@
+// # String Exercises
+//
+// 1. console.log the first character of a string.
+let str = "water";
+console.log(str[0]);
+
+// 2. console.log the length of a string.
+let fud = "lasagna";
+console.log(fud.length);
+
+// // 3. console.log the last character of any string.
+let flower = "tulip";
+console.log(flower[flower.length -1]);
+
+// 4. Create a code block that takes a single string variable, and returns a copy of the string with the last letter capitalized. For example:
+//
+// ```js
+// 'jimmy'
+// // => 'jimmY'
+// ```
+let word = "chips";
```

Comment:
This is not a good code as it is hard coding. If I change word to "chips&fish" then it would not work. Is there way to fix this by changing (word.slice(0, 4)) somehow? hint is using string length.